prime-router/metadata/schemas/covid-19.schema

```diff
@@ -351,7 +351,7 @@ elements:
     type: TABLE
     pii: false
     table: fips-county
-    tableColumn: County
+    tableColumn: FIPS
     mapper: lookup(ordering_provider_state, ordering_provider_county)
```

Comment:
The `ordering_provider_county_code` had was always outputting the county name instead of the FIPS code.